src/SDK/Language/Node.php

```diff
@@ -138,7 +141,8 @@ public function getParamExample(array $param): string
                     $output .= "'{$example}'";
                     break;
                 case self::TYPE_FILE:
-                    $output .= "InputFile.fromPath('/path/to/file', 'filename')";
+                    $blob = file_get_contents('path/to/file');
+                    $output .= "Payload.fromFile(" . $blob . ", 'filename')";
```

Comment:
This is for docs I believe. Let's update it with proper example once we have final decision from Christy